numba/cuda/stubs.py

```diff
@@ -656,3 +657,40 @@ class habs(Stub):
         Returns the fp16 result of the absolute value.
 
         """
+
+
+#-------------------------------------------------------------------------------
+# vector types
```

Comment:
Should all of the implementation of the generation of `_vector_type_stubs` go in a function to make it easier to avoid leaking into the module namespace? e.g. `cuda.stubs.name` is `"double"` and `cuda.stubs.nelem` is `4` after import at present.